src/models/AppState.js

```diff
@@ -321,6 +334,15 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
           return res._links.next.href;
         }
       },
+      'profileSchemaAttributes': {
```

Comment:
we have a `policy` attribute, that might contain already what you need and we can avoid this derived property
If it does not, this should be dependent on `lastAuthResponse` I think. Also the name is a little misleading since it contains the full registration object, not just the schema attributes.

Reply:
ah, good point. I'll re-use that.